StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -110,18 +110,13 @@ extension STPAPIClient {
 
         let billingParams = billingDetails.consumersAPIParams
 
-        var card = STPFormEncoder.dictionary(forObject: cardParams)["card"] as? [AnyHashable: Any]
-        card?["cvc"] = nil // payment_details doesn't store cvc
```

Comment:
We were previously niling out this field, but no longer seem to be doing so.

Reply:
Instead, we're using `consumersAPIParams` and not adding the `cvc` field to begin with.